run.py

```diff
@@ -106,6 +123,30 @@ def config_validator():
         app.config['TESTING'] = True
         print("Starting server in ~*TEST MODE*~")
         app.run(host='localhost')
+    elif args.backup:
+        if os.path.isfile(os.path.join(os.getcwd(), 'issues.db')):
+            issue_engine = create_engine('sqlite:///' + os.path.join(
+                app.config['BASE_DIR'], 'issues.db'))
+            issue_session_maker = sessionmaker(autocommit=False,
+                                               autoflush=False,
+                                               bind=issue_engine)
+            issue_db = scoped_session(issue_session_maker)
+            # Take a backup if issues.db has data dump.
+            if issue_db().execute('select count(*) from webcompat_issues').fetchall()[0][0] > 0:
+                if not os.path.exists(app.config['BACKUP_DEFAULT_DEST']):
+                    print 'Backup db folder does not exist'
+                    os.makedirs(app.config['BACKUP_DEFAULT_DEST'])
+                else:
+                    print 'Backup db folder exists'
+                time_stamp = time.strftime('%Y-%m-%dT%H-%M-%S', time.localtime())
+                issue_backup_db = 'issues_' + time_stamp + '.db'
+                os.rename(os.getcwd() + '/issues.db', app.config['BACKUP_DEFAULT_DEST'] + issue_backup_db)
```

Comment:
The [line #44](https://github.com/deepthivenkat/webcompat.com/blob/1e6f8c3ef4f590342ba36db38b622baa329c9f04/run.py#L44) gets executed irrespective of the arguments given to run.py.

This line recreates issues.db with old schema. Now I need to write code to read alter commands for webcompat_issues table from a configured location and execute all of them. Then we will have the updated schema. I will write for this code in a separate issue.

Since issues.db gets recreated every time the python script is run, I am removing the check if issues.db file exists. I am also removing the FILE_NOT_FOUND error since the case will never happen